prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+document.addEventListener('DOMContentLoaded', function () {
+
+});
+
+console.log('content loaded!');
+
+window.addEventListener('mousemove', function () {
+
+});
+
+console.log('mouse is moving!');
+
```

Comment:
Hi Julie! Your console logs should be inside of your event listener functions so they trigger when those things happen